Makefile

```
@@ -1,5 +1,5 @@
 # commit or branch for the extern/filecoin-ffi dependency
-filecoin_ffi_branch = 5d00bb4365a97890
+filecoin_ffi_branch = 32afd6e1f1419b6bb7d0f4b3944287fde593ca64
```

```suggestion
filecoin_ffi_branch = 32afd6e1f1419b6b
```

currently we're strictly doing branches, not tags, so we have to go with the weird filecoin-ffi tag-branch names